csfle/node/kmip/reader/insert_encrypted_document.js

```diff
@@ -62,6 +62,10 @@ const schema = {
   },
 };
 
+var patientSchema = {};
```

Comment:
See https://github.com/10gen/docs-mongodb-internal/pull/1745#discussion_r959903032 for more information